apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -343,25 +343,27 @@ const Arrow: React.FC = () => {
 
 const AutoSize = () => {
   const styles = useStyles();
+  const [overflowBoundary, setOverflowBoundary] = React.useState<HTMLDivElement | null>(null);
```

Comment:
This test actually never worked, `overflow: 'hidden'` would make this test always pass